### ISSUE TITLE
Print the framework shortname in kpm list output 

### ISSUE THREAD

**Issue by Praburaj** [aspnet/XRE#1099]
```
[Target framework Asp.Net]
EntityFramework 7.0.0-beta3-11988
    -> Microsoft.Framework.CodeGeneration.EntityFramework 1.0.0-beta3-11512
...
...
```

Currently the output of `kpm list` is something like this. It will be useful to have the framework shortnames like `aspnet50` or `aspnetcore50` listed in the output like to enable searching through long output graphs. 

```
[Target framework Asp.Net (aspnet50)]
```

**Comment by troydai** [comment]
I don't think it's doable. https://github.com/aspnet/XRE/blob/dev/src/Microsoft.Framework.Runtime/NuGet/Utility/VersionUtility.cs#L178

**Comment by davidfowl** [comment]
Of course it is. What do you mean?

**Comment by troydai** [comment]
https://github.com/aspnet/XRE/blob/dev/src/Microsoft.Framework.Runtime/NuGet/Utility/VersionUtility.cs#L1140
The `aspnet50` is mapped to `Asp.Net`. But the dictionary is private. When the `DependencyListCommand` consumes the `FrameworkName` the convert is already done. How do I reverse the name back to `aspnet50`?

**Comment by davidfowl** [comment]
GetShortframeworkname

**Comment by troydai** [comment]
Ah, ok.

**Comment by troydai** [comment]
https://github.com/aspnet/XRE/pull/1198

**Comment by troydai** [comment]
Fixed in e75955245d966eef5d95b8c023af9e9276415a51
